src/components/Search/SearchView/SearchView.tsx

```diff
@@ -65,7 +65,7 @@ function SearchView(props: SearchViewProps) {
           <Grid item xs={1} />
         </Grid>
         <Grid item xs={12}>
-          <SelectedRevisionsTable />
+          {searchResults.length > 0 && <SelectedRevisionsTable />}
```

Comment:
this should be checking the length of selectedRevisions rather than search results

Reply:
You'll need to add a prop similar to this for selectedRevisions: https://github.com/mozilla/perfcompare/blob/c73eb07f7413eb873b5d6ad9ca340b86fd00d945/src/components/Search/SearchView/SearchView.tsx#L81

Reply:
true, it seemed strange at first cos it was still rendering, but makes sense. thanks!